data/json/items/armor/helmets.json

```diff
@@ -449,9 +458,11 @@
     "material": [ "plastic", "nomex" ],
     "symbol": "[",
     "color": "dark_gray",
-    "covers": [ "HEAD", "MOUTH" ],
-    "coverage": 100,
-    "encumbrance": 25,
+    "armor_portion_data": [
+      { "covers": [ "head" ], "coverage": 100, "encumbrance": 25 },
+      { "covers": [ "eyes" ], "coverage": 100, "encumbrance": 5 },
+      { "covers": [ "mouth" ], "coverage": 100, "encumbrance": 10 }
```

Comment:
I've never actually worn a motorcycle helmet in this style, so feel free to ignore, but I'd probably bump up the encumbrance by 5 on the mouth.